lib/email_doc/document.rb

```diff
@@ -5,22 +5,22 @@ module EmailDoc
   class Document
     extend Forwardable
 
-    attr_reader :mail, :context
+    attr_reader :mail, :context, :ex
```

Comment:
`example` is not available from within an `it` block. so I use `ex` in render methods.
However, as you say, `ex` is It would associate multiple meanings(e.g. `Exception`, `Example`, `Exercise `). 😕  I'll think about it in the next chance!